tools/StaticAnalysis/Exceptions/Az.NetworkCloud/SignatureIssues.csv

```diff
@@ -0,0 +1,83 @@
+"Module","ClassName","Target","Severity","ProblemId","Description","Remediation"
```

Comment:
Please remove rows whose severity > 1

Reply:
We saw some error with Sev>1 , Can you please suggest how we could resolve this ?

"Az.NetworkCloud","New-AzNetworkCloudIpAddressPoolObject","New-AzNetworkCloudIpAddressPoolObject","2","8010","**New-AzNetworkCloudIpAddressPoolObject Changes the ConfirmImpact but does not set the SupportsShouldProcess property to true in the cmdlet attribute.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue**"

Reply:
Hi @priyamshet , these messages should be warning rather than error messages. We can ignore them after double confirm

Reply:
Thank @YanaXu  for clarifying. feedback incorporated